pkg/manager/impl/validation/validation.go

```diff
@@ -196,6 +203,11 @@ func validateParameterMap(inputMap *core.ParameterMap, fieldName string) error {
 	return nil
 }
 
+func ValidateProjectUpdateRequestLabelsAlphanumeric(request admin.Project) error {
+	// TODO(kosigz): implement this
```

Comment:
`validation.IsDNS1123Label()` should be sufficient!

Reply:
Thanks! 👍 On it.